app/bundles/SmsBundle/Tests/Helper/ReplyHelperTest.php

```diff
@@ -65,7 +66,7 @@ public function testFoundContactsDispatchEvent()
 
     public function testHandlerResponseIsReturnedIfResponseInterface()
     {
-        $handler = $this->createMock([CallbackInterface::class, ResponseInterface::class]);
+        $handler = $this->createMock(CallbackResponseInterface::class);
```

Comment:
Instead of creating new interface that should be used only in 1 test I'd rather do something like this:
```php
$handler = new class implements CallbackInterface, ResponseInterface {
    public function getResponse() {
        return new Response('hi');
    }
    public function getContacts(Request $request) {
        return new ArrayCollection([new Lead()]);
    }

    public function getMessage(Request $request) {
        return '';
    }

    public function getTransportName() {
        return '';
    }
}
```

Reply:
Thanks @escopecz, creative suggestion 😊 Done in https://github.com/mautic/mautic/pull/9244/commits/182b5c73dc6ae5b9226e0bfcb8317edcf40698f5. If you please could go ahead and approve + **squash and merge** (we recently started squashing for a cleaner Git history), that'd be great! Thanks

![image](https://user-images.githubusercontent.com/17739158/99947265-0483bd00-2d78-11eb-8cac-bed6b3d7c6ab.png)

Reply:
Will do. It has some CS issues though

Reply:
Oops, sorry. Just fixed @escopecz